salt/utils/files.py

```diff
@@ -416,11 +416,13 @@ def flopen(*args, **kwargs):
     '''
     Shortcut for fopen with lock and context manager.
     '''
-    with fopen(*args, **kwargs) as f_handle:
+    filename = args.pop(0)
+    vars = 'wa'
```

Comment:
Could we use something else instead of `vars`? it shadows a built-in.

Reply:
yeah